### PR TITLE
feat: LSDV-5296: Scroll container max height

### PR DESCRIPTION
**Pull request by Travis1282**
### PR fulfills these requirements
- [X] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [X] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [X] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [X] Frontend

### Describe the reason for change
Add maxheight as a config attribute with a default of 500px and style scrolling window

#### What does this fix?
N/A

#### What is the new behavior?
The config attribute defines the maximum height of the scrolling window 

#### What is the current behavior?
it takes up as much space as it's contents will allow 

#### What libraries were added/updated?
none

#### Does this change affect performance?
no

#### Does this change affect security?
no

#### What alternative approaches were there?
we could have used flex box or javascript to calc the available space 

#### What feature flags were used to cover this change?
fflag_feat_front_lsdv_e_278_contextual_scrolling_short

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [X] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: src/tags/object/Paragraphs/model.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ const isFFLsdv4711 = isFF(FF_LSDV_4711);
  * @param {none|dialogue} [layout=none]  - Whether to use a dialogue-style layout or not
  * @param {string} [nameKey=author]      - The key field to use for name
  * @param {string} [textKey=text]        - The key field to use for the text
+ * @param {string} [maxheight]           - Maximum height of the paragraph container
```
Comment (hlomzik) [inline src/tags/object/Paragraphs/model.js:51]: camelCase and default value

File: src/tags/object/Paragraphs/Paragraphs.module.scss [hunk 2]
```diff
@@ -59,6 +59,14 @@
   cursor: text;
 }
 
+.scroll_container {
+  position: relative;
+  overflow: auto;
+  counter-reset: phrase;
+  border: 1px solid rgba(137, 128, 152, 0.16);
```
Comment (hlomzik) [inline src/tags/object/Paragraphs/Paragraphs.module.scss:66]: can we move this to constants?

File: src/utils/feature-flags.ts [hunk 3]
```diff
@@ -262,6 +262,13 @@ export const FF_LSDV_4998 = 'fflag_fix_front_lsdv_4998_missed_dynamic_children_0
  */
 export const FF_PROD_309 = 'fflag_feat_front_prod_309_choice_hint_080523_short';
 
+/**
+ * Contextual scrolling of Paragraph segments with Audio V0
+ * @link https://app.launchdarkly.com/default/production/features/fflag_feat_front_lsdv_e_278_contextual_scrolling_short/targeting
+ */
+
+export const FF_LSDV_E_278 = 'fflag_feat_front_lsdv_e_278_contextual_scrolling_short';
```
Comment (hlomzik) [inline src/utils/feature-flags.ts:270]: please, reorder it to be sorted

**Review by hlomzik — APPROVED** [review]

**Review by hlomzik — CHANGES_REQUESTED** [review]

File: src/utils/feature-flags.ts (hunk 3 above)
Comment (Travis1282) [reply src/utils/feature-flags.ts:270]: ah thanks

File: src/tags/object/Paragraphs/HtxParagraphs.js [hunk 4]
```diff
@@ -399,23 +402,47 @@ class HtxParagraphsView extends Component {
...
+    const totalSpace = mainContentView?.offsetHeight || 0;
+    const filledSpace = annotationView?.offsetHeight || 0;
+    const containerHeight = this.myRef.current?.offsetHeight || 0;
+    const viewPadding = parseInt(window.getComputedStyle(mainContentView)?.getPropertyValue('padding-bottom')) || 0;
+    const annotationPadding = parseInt(window.getComputedStyle(annotationView)?.getPropertyValue('padding-bottom')) || 0;
+    const height = totalSpace - (filledSpace - containerHeight) - viewPadding - annotationPadding;
+    const minHeight = 100;
+
+    this.setState(
+      { scrollContainerHeight: height < minHeight ? minHeight : height },
```
[the remark this answers is not in the document]
Comment (Travis1282) [reply src/tags/object/Paragraphs/HtxParagraphs.js:417]: good thinking

File: src/tags/object/Paragraphs/model.js (hunk 1 above)
Comment (Travis1282) [reply src/tags/object/Paragraphs/model.js]: ok, isn't the default flase here?

**Review by hlomzik — APPROVED** [review]

**Pull request merged**